<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="FunBtn_00">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>无限阳光</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>卡槽冷却</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_02">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>智慧树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_03">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>无限金币</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_04">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>秒杀博士</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_05">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>全屏轰炸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_06">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>植物无敌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_07">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>僵尸无敌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunBtn_08">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="text">
     <string>罐子透视</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="FuncBox_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>460</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="title">
     <string>种植相关</string>
    </property>
    <widget class="QPushButton" name="FunBtn_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>僵尸放置</string>
     </property>
    </widget>
    <widget class="QLabel" name="L1_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>僵尸选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="FunBtn_09">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>植物种植</string>
     </property>
    </widget>
    <widget class="QLabel" name="L1_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>植物选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="YComBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ZombiesComBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>普通僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>旗子僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>路障僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>撑杆僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>铁桶僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>读报僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>铁门网僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>橄榄球僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>舞者僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>伴舞僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>鸭子救生圈僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>潜水僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>雪橇车僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>雪橇车僵尸小队</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>海豚骑士僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小丑僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>气球僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>矿工僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蹦蹦僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>雪人僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>飞贼僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>梯子僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>投石车僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>巨人僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小鬼僵尸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>僵尸博士</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="L1_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>列</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="L1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>行</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="XComBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="PlantsComBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>豌豆射手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>向日葵</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>樱桃炸弹</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>坚果</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>土豆雷</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>寒冰射手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>大嘴花</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双重射手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小喷菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>阳光菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>大喷菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>咬咬碑</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>迷糊菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>胆小菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>冰川菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>末日菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>莲叶</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>窝瓜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>三重射手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>缠绕水草</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>火爆辣椒</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>地刺</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>火炬树状</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高坚果</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>水兵菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>路灯花</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>仙人掌</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>三叶草</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双向射手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>星星果</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>南瓜头</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>磁力菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>卷心菜投手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>花盆</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>玉米投手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>咖啡豆</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>大蒜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>萝卜伞</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>金盏花</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西瓜投手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>机枪投手</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双胞向日葵</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>多嘴小喷菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>猫尾草</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>冰西瓜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>吸金菇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>钢地刺</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>玉米加农炮</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FuncBox_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>460</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="title">
     <string>变速功能</string>
    </property>
    <widget class="QPushButton" name="FunBtn_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QSlider" name="SpeedSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lspeed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FuncBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>460</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
font-family:songti;</string>
    </property>
    <property name="title">
     <string>反反作弊驱动</string>
    </property>
    <widget class="QPushButton" name="FunBtn_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;
font-family:songti;</string>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="StartMenu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="InjectAction"/>
    <addaction name="UnloadAction"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="StartMenu"/>
  </widget>
  <action name="InjectAction">
   <property name="text">
    <string>注入</string>
   </property>
  </action>
  <action name="UnloadAction">
   <property name="text">
    <string>卸载</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
